docs/topics/db/transactions.txt

```diff
@@ -176,6 +176,27 @@ Django provides a single API to control database transactions.
         If you catch exceptions raised by raw SQL queries, Django's behavior
         is unspecified and database-dependent.
 
+    .. note::
+
+        You model state will not be reverted whenever a transaction rollback
```

Comment:
"You**r** model state ..." or "The model state ..."
and
"... unless manually reverted::"